### PR TITLE
feat: enable source maps in production

### LINKED ISSUE #776: Enable source maps in production
For easier debugging. Currently, it looks like

Screenshot 2024-05-11 at 13 05 24

(Chrome@124 at https://aescan.io/)

This won't affect overall performance. We don't care about source code exposure because we are already open-source 🤡 

May resolve https://github.com/aeternity/aescan/issues/323 and https://github.com/aeternity/aescan/issues/252 as well

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #776
resolves https://github.com/aeternity/aescan/issues/323
taken inspiration from this https://gearboxgo.com/articles/web-application-development/setting-up-sentry-with-nuxt-3

## Demo
Not sure how should I verify this. I guess we need to wait for merge and deployment, then verify it

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] enable source maps in production

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -79,6 +81,17 @@ export default defineNuxtConfig({
...
         },
       },
+      sentryVitePlugin({
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        org: 'sentry',
+        project: 'aescan-develop',
+        url: 'https://sentry.dev.service.aepps.com/',
+      }),
+      sentryVitePlugin({
+        org: 'sentry',
```
Comment (michele-franchi) [inline nuxt.config.ts:91]: Is it correct that there's no `authToken` here?

**Review by michele-franchi — COMMENTED** [review]
Please check my comment.

And, is the env variable NUXT_PUBLIC_SENTRY_AUTH_TOKEN already configured?

**Comment by janmichek** [comment]
> Please check my comment.
> 
> And, is the env variable NUXT_PUBLIC_SENTRY_AUTH_TOKEN already configured?

Yes, it's set on individual servers

File: nuxt.config.ts (hunk 1 above)
Comment (janmichek) [reply nuxt.config.ts:92]: Right, fixed

**Review by michele-franchi — APPROVED** [review]
LGTM

**Pull request merged**